frontend_v2/src/app/components/analytics/host-analytics/host-analytics.component.scss

```diff
@@ -63,4 +63,6 @@ h4.analytics-details-head {
   }
 }
 
-
+.ev-md-container {
+  margin-top: -20px;
```

Comment:
Umm, seems like something is off here. We should not be using the `-20px` or any px related css as @Shekharrajak pointed out in the comment.